libs/ui-toolkit/src/components/form-group/form-group.tsx

```diff
@@ -4,32 +4,48 @@ import type { ReactNode } from 'react';
 interface FormGroupProps {
   children: ReactNode;
   label?: string;
-  labelFor?: string;
   labelAlign?: 'left' | 'right';
+  labelDescription?: string;
   className?: string;
+  hasError?: boolean;
 }
 
 export const FormGroup = ({
   children,
   label,
-  labelFor,
```

Comment:
chore (blocking): Lets keep this please. We need a way to accessibly link labels to input elements

Reply:
@mattrussell36 Implicit association is accessible. I've made a codepen to demo to you - https://codepen.io/samkeen/pen/eYVpLjg/b12ef60491cc90e2ee048f3c62477a5d

Enable voiceover on your mac and focus on each of the three inputs. I know older browsers struggled with implicit association, but it's not a problem I've seen in a while now.

Reply:
@mattrussell36 scratch that. Works well on FF but seeing issues in Safari, which is bonkers. I'll re-add it.